eng/Version.Details.xml

```diff
@@ -379,7 +379,7 @@
     </Dependency>
     <!-- Necessary for source-build. This allows the package to be retrieved from previously-source-built artifacts
          and flow in as dependencies of the packages produced by runtime. -->
-    <Dependency Name="Nuget.ProjectModel" Version="6.2.2">
+    <Dependency Name="Nuget.ProjectModel" Version="6.2.4">
```

Comment:
This should be updated using [darc](https://github.com/dotnet/arcade/blob/main/Documentation/Darc.md) to get both the version and the SHA updated.

Reply:
nit: you can also find the SHA in the nuspec from nuget.org.

Reply:
Do you happen to know the darc command that should be used to update this or the channel that this dependency flows from?

Reply:
I tried using the update-dependencies command targetting specifically the Nuget.ProjectModel dependency. The version updated but the Sha was not updated. Do you know if there are extra steps or specific commands to run to get the Sha to update also? Thanks!

Reply:
Thanks! That worked Viktor. Github hadn't loaded your message before I had tried the other stuff and sent those messages.